README.md

```diff
@@ -1,2 +1,30 @@
-# pallene-tracer
-Pallene Tracer is a mechanism and protocol for Lua C modules to have proper tracebacks
+<h1 align="center">The Pallene Tracer</h1>
+
+Pallene Tracer is a project providing protocols and underlying mechanism aiming to enable Lua C modules to have proper function call-traceback, without touching a single code in Lua internals!
+
+## Installing Pallene Tracer
+
+### Prerequisites
+
+You need to have Lua installed in your system in any prefix.
+
+### Procedure
+
+To install Pallene Tracer in `/usr/local` prefix, against system Lua with `/usr` prefix: 
```

Comment:
I suggest that we separate the make into two steps, one to compile and one to install. That way we can talk about compilation flags (LUA_PREFIX, LDFLAGS) separately from installation flags (PREFIX)